Description: lintian reports hyphen misuse in manpages. This patch fixes those
 reported.
Author: Paul Gevers <elbrus@debian.org>
Forwarded: http://bugs.motifzone.net/show_bug.cgi?id=1588
Last-reviewed: 06 Feb 2013

--- a/doc/man/man1/mwm.1
+++ b/doc/man/man1/mwm.1
@@ -904,7 +904,7 @@
 resizeCursorsResizeCursorsT/FT
 transientDecorationTransientDecorationstringmenu title
 transientFunctionsTransientFunctionsstringT{
--minimize-maximize
+\-minimize-maximize
 T}
 useIconBoxUseIconBoxT/FF
 .TE
@@ -941,7 +941,7 @@
 screen size of the icon box window depends on the iconImageMaximum (size)
 and \fIiconDecoration\fP resources\&. The default value for size
 is (6 * iconWidth + padding) wide by (1 * iconHeight + padding) high\&. The
-default value of the location is +0 -0\&.
+default value of the location is +0 \-0\&.
 .IP "\fIiconBoxName\fP\ (class\  \fIIconBoxName\fP)" 10
 This resource specifies the name that is used to look up icon box resources\&.
 The default name is iconbox\&.
@@ -1059,7 +1059,7 @@
 This resource is used to indicate which window management functions
 are applicable (or not applicable) to transient windows\&. The function specification
 is exactly the same as for the \fIclientFunctions\fP (client
-specific) resource\&. The default value for this resource is -minimize -maximize\&.
+specific) resource\&. The default value for this resource is \-minimize \-maximize\&.
 .IP "" 10
 An application can also specify which functions \fBmwm\fP
 should apply to its windows\&. If it does so, \fBmwm\fP applies
@@ -1176,7 +1176,7 @@
 titleTitle bar (includes border)\&.
 .TE
 .PP
-Examples: \fIMwm*XClock\&.clientDecoration: -resizeh -maximize\fP This removes the resize handles and maximize button from XClock
+Examples: \fIMwm*XClock\&.clientDecoration: \-resizeh \-maximize\fP This removes the resize handles and maximize button from XClock
 windows\&. \fIMwm*XClock\&.clientDecoration: menu minimize border\fP This does the same thing as above\&. Note that either \fImenu\fP or \fIminimize\fP implies
 \fItitle\fP\&.
 .IP "\fIclientFunctions\fP\ (class\  \fIClientFunctions\fP)" 10
--- a/doc/man/man4/mwmrc.4
+++ b/doc/man/man4/mwmrc.4
@@ -84,7 +84,7 @@
 be specified from the command line:
 .PP
 .nf
-\f(CW/usr/X11R6/bin/X11/mwm -xrm "mwm*configFile: mymwmrc"\fR
+\f(CW/usr/X11R6/bin/X11/mwm \-xrm "mwm*configFile: mymwmrc"\fR
 .fi
 .PP
 .SS "Resource Types"
--- a/doc/man/man3/XmFontList.3
+++ b/doc/man/man3/XmFontList.3
@@ -119,7 +119,7 @@
 without a font list entry tag;
 .PP
 .nf
-\f(CW*fontList:  -Adobe-Times-Medium-I-Normal--10*\fR
+\f(CW*fontList:  \-Adobe\-Times\-Medium\-I\-Normal\-\-10*\fR
 .fi
 .PP
 .PP
@@ -138,9 +138,9 @@
 and an explicit font list entry tag\&.
 .PP
 .nf
-\f(CW*fontList: -Adobe-Courier-Bold-R-Normal--25-180-100-100-M-150;\
--JIS-Fixed-Medium-R-Normal--26-180-100-100-C-240;\
--JIS-Fixed-Medium-R-Normal--26-180-100-100-C-120:MY_TAG\fR
+\f(CW*fontList: \-Adobe\-Courier\-Bold\-R\-Normal\-\-25\-180\-100\-100\-M\-150;\
+\-JIS\-Fixed\-Medium\-R\-Normal\-\-26\-180\-100\-100\-C\-240;\
+\-JIS\-Fixed\-Medium\-R\-Normal\-\-26\-180\-100\-100\-C\-120:MY_TAG\fR
 .fi
 .PP
 .PP
--- a/doc/man/man3/XmGetPixmapByDepth.3
+++ b/doc/man/man3/XmGetPixmapByDepth.3
@@ -245,8 +245,8 @@
 \f(CW/* XBM file */
 #define plaid_width 22
 #define plaid_height 22
-#define plaid_x_hot -1
-#define plaid_y_hot -1
+#define plaid_x_hot \-1
+#define plaid_y_hot \-1
 static char plaid_bits[] = {
    0x75, 0xfd, 0x3f, 0xaa, 0xfa, 0x3e, 0x75, 0xfd, 0x3f, 0xaa, 0xfa, 0x3e,
    0x75, 0xfd, 0x3f, 0xff, 0x57, 0x15, 0x75, 0xfd, 0x3f, 0xaa, 0xfa, 0x3e,
--- a/doc/man/man3/XmList.3
+++ b/doc/man/man3/XmList.3
@@ -182,7 +182,7 @@
 A value of 1 transfers the first item in the List; a value of 2
 transfers the second item; and so on\&.
 If the entire contents of the List are being transferred, the value is
--1\&.
+\-1\&.
 .PP
 As a source of data, List supports the following targets and associated
 conversions of data to these targets:
--- a/doc/man/man3/XmScreen.3
+++ b/doc/man/man3/XmScreen.3
@@ -300,7 +300,7 @@
 startup time, either by placing it within a defaults file or by using the
 \fB-xrm\fP command line argument\&. For example:
 .IP "" 10
-\fBmyProg -xrm "*menuCursor: arrow"\fP
+\fBmyProg \-xrm "*menuCursor: arrow"\fP
 .IP "" 10
 The menu cursor can also be selected in the program through
 the function \fBXmSetMenuCursor\fP\&.
--- a/doc/man/man3/XmTabListCopy.3
+++ b/doc/man/man3/XmTabListCopy.3
@@ -71,7 +71,7 @@
 Specifies where to start copying\&. A value of 0 (zero) indicates begin
 at the beginning, a value of 1 indicates to skip the first tab, and so
 on\&. A negative indicates to begin counting backwards from the end\&.
-A value of -1 indicates to start copying from the last tab\&.
+A value of \-1 indicates to start copying from the last tab\&.
 .IP "\fIcount\fP" 10
 Specifies the number of tabs to copy\&. A value of 0 (zero) indicates
 to copy all elements from the starting point to the end (beginning if
--- a/doc/man/man3/XmTabListInsertTabs.3
+++ b/doc/man/man3/XmTabListInsertTabs.3
@@ -83,7 +83,7 @@
 value of 1 makes it the second tab, and so on\&. If \fIposition\fP is
 greater than the number of tabs in \fIoldlist\fP, then the tabs will
 be inserted at the end\&. If \fIposition\fP is negative, the count will
-be backwards from the end\&. A value of -1 makes the first new tab
+be backwards from the end\&. A value of \-1 makes the first new tab
 the last tab, and so on\&.
 .SH "RETURN"
 .PP
--- a/doc/man/man5/UIL.5
+++ b/doc/man/man5/UIL.5
@@ -1211,7 +1211,7 @@
 3\&.1415E-2 (equals \&.031415)
 T}2\&.87 e6 (embedded blanks)
 T{
--6\&.29e7 (equals -62900000)
+\-6\&.29e7 (equals \-62900000)
 T}2\&.0e100 (out of range)
 .TE
 .PP
@@ -1322,7 +1322,7 @@
 You can use parentheses to override the normal precedence of operators\&.
 In a sequence of unary operators, the operations are performed in
 right-to-left
-order\&. For example, \fB- + -A\fP is equivalent to \fB-(+(-A))\fP\&.
+order\&. For example, \fB- + \-A\fP is equivalent to \fB\-(+(\-A))\fP\&.
 In a sequence of binary operators of the same precedence, the operations
 are performed in left-to-right order\&. For example, \fBA*B/C*D\fP is
 equivalent to \fB((A*B)/C)*D\fP\&.
